recipes/virtual-libjpeg/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+    "9.0":
```

Comment:
```suggestion
    "virtual":
```

Reply:
jpeg interface have many version 7.0, 8.0 and 9.0 (the last one) and we provide a virtual package for the version 9. I think we should use the version of the API that this virtual package provide.

Reply:
So we should also provide a mapping between libjpeg 9 API and libjpeg-turbo and mozjpeg compatible versions.